types/react-addons-linked-state-mixin/package.json

```diff
@@ -6,10 +6,10 @@
         "http://facebook.github.io/react/"
     ],
     "dependencies": {
-        "@types/react": "*"
+        "@types/react": "*",
```

Comment:
Need this so that module augmentation applies. Otherwise we get failures like https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/7408220194/job/20155995314?pr=68096.